example/ansible/playbook.yml

```
@@ -24,3 +24,23 @@
       command: nmcli radio wifi off
       become: true
       when: "'NetworkManager' in ansible_facts.packages"
+
+    - name: test ssh
+      ansible.builtin.wait_for:
+        host: "{{ inventory_hostname }}"
+        port: 22
+      delegate_to: localhost
+
+    - name: test webpage access
+      action: uri url=https://www.example.com return_content=yes
```

Please use modern ansible syntax; this format is the legacy deprecated format.

```suggestion
      uri:
        url: https://www.example.com
        return_content: yes
```